src/plugins/ExtendedProfileSampling.cpp

```diff
@@ -0,0 +1,862 @@
+/*
+ * ExtendedProfileSampling.cpp
+ *
+ *  Created on: Feb 2022
+ *      Author: homes
+ */
+
+#include "ExtendedProfileSampling.h"
+
+#include "particleContainer/ParticleContainer.h"
+#include "particleContainer/adapter/ParticlePairs2PotForceAdapter.h"
+#include "utils/Random.h"
+#include "Simulation.h"
+
+#include <math.h>
+
+
+ExtendedProfileSampling::ExtendedProfileSampling()
+    :
+    // Control: general
+    _binwidth(1.0f),
+    _startSampling(0ul),
+    _writeFrequency(10000ul),
+    _stopSampling(1000000000ul),
+    _singleComp(false),
+
+    // Higher moments
+    _sampleHigherMoms(false),
+
+    // Control: chemical potential
+    _sampleChemPot(false),
+    _lattice(true),
+    _factorNumTest(4.0f),
+    _samplefrequency(50ul)
```

Comment:
When these values are not specific to this constructor but should be that way everywhere set them at the declaration of the member variables in the header.
Since I only see one constructor this should apply to all of them. Make sure everything is initialized after the constructor is called